actionview/CHANGELOG.md

```diff
@@ -11,7 +11,16 @@
 
 ## Rails 5.1.6.2 (March 11, 2019) ##
 
-*   No changes.
+*   Only accept formats from registered mime types
+
+    A lack of filtering on mime types could allow an a attacker to read
```

Comment:
Looks like there's a typo here `an a`

Reply:
@sikachu Opened a PR for the fix. https://github.com/rails/rails/pull/35722